docs/Ordinal.md

```diff
@@ -1,13 +1,41 @@
 # Ordinal Support
 
-It is possible to use an ordinal as part of the collateral of a DLC.
-To do so, an [`OrdDescriptor`](../dlc-manager/src/contract/ord_descriptor.rs) must be used.
-This descriptor contains information about the ordinal being included in the DLC (location in the blockchain and the transaction that includes is), as well as information about the event upon which the DLC is based.
+This library now supports the locking of an ordinal as part of the collateral of a DLC.
+
+Notes:
+* An ordinal is inscribed into a single satoshi. However, that satoshi is often part of a larger UTXO. DLCs lock UTXOs, so please keep that context in mind.
```

Comment:
I'm not an expert, but I don't think an ordinal is incribed. Some content can be "inscribed" into a single stats, but AFAIU an ordinal is the number associated with the sats.

Reply:
Also I'm not sure what you're trying to say here tbh.

Reply:
Again I'm trying to give context for the whole effort. However, if you prefer to keep the document closer to as you had it before, I'll move some of this conversation outside to our own docs page. What do you think?

Reply:
Yeah maybe that's better, I don't think this is the place where people should learn about ordinals tbh (I'd expect that someone who wants to use this feature of the library would already know about it).

Reply:
I agree. If someone's so deep in the rust-dlc code, they should have some background on the various technologies which brought them there.